.github/workflows/release.yml

```diff
@@ -55,3 +55,15 @@ jobs:
         if: >- # "create" workflows run separately from "push" & "pull_request"
           github.event_name == 'release'
         uses: pypa/gh-action-pypi-publish@release/v1
+
+  forum_post:
```

Comment:
That is a different JOB! obviously that you do not have python setup to it. Now the question is why you wanted a different job?